theme.config.tsx

```diff
@@ -1,51 +1,49 @@
-import { useRouter } from 'next/router';
-import { DocsThemeConfig, useConfig } from 'nextra-theme-docs';
-import Banner from './components/banner';
-import { NavCTA, TocCTA } from './components/cta';
-import Footer from './components/footer';
-import { Logo } from './components/logo';
-import { Flexsearch } from './components/nextra/Flexsearch';
+import { useRouter } from "next/router";
+import { DocsThemeConfig, useConfig } from "nextra-theme-docs";
+import Banner from "./components/banner";
+import { NavCTA, TocCTA } from "./components/cta";
+import Footer from "./components/footer";
+import { Logo } from "./components/logo";
+import { Flexsearch } from "./components/nextra/Flexsearch";
 
 const config: DocsThemeConfig = {
   logo: Logo,
-  logoLink: 'https://authzed.com',
-  project: { link: 'https://github.com/authzed/spicedb' },
-  useNextSeoProps() {
+  logoLink: "https://authzed.com",
+  project: { link: "https://github.com/authzed/spicedb" },
+  head: () => {
     const { asPath } = useRouter();
-    const { title, frontMatter } = useConfig();
+    const { title: titleContent, frontMatter } = useConfig();
     const desc =
       frontMatter.description ||
-      'Welcome to the SpiceDB and AuthZed docs site.';
-
-    const defaultConfig = {
-      defaultTitle: 'AuthZed Docs',
-      titleTemplate: '%s – AuthZed Docs',
-      description: desc,
-      canonical: `https://authzed.com${
-        process.env.NEXT_PUBLIC_BASE_DIR ?? ''
-      }${asPath}`,
-      openGraph: {
-        title,
-        description: desc,
-      },
-    };
-
-    return defaultConfig;
-  },
-  head: () => {
+      "Welcome to the SpiceDB and AuthZed docs site.";
+    const resolvedTitle = titleContent
+      ? `${titleContent} - Authzed Docs`
+      : "Authzed Docs";
     return (
       <>
+        <title>{resolvedTitle}</title>
+        <meta name="description" content={desc} />
+        <meta property="og:title" content={resolvedTitle} />
+        <meta property="og:description" content={desc} />
+        <link
+          rel="canonical"
+          href={`https://authzed.com${
+            process.env.NEXT_PUBLIC_BASE_DIR ?? ""
+          }${asPath}`}
+        />
```

Comment:
This was one of the more meaningful changes - rather than having a separate hook/property for SEO stuff, nextra v3 just uses `head` tag stuff. Please check my work here.